packages/shared/CheckStringCoercion.js

```diff
@@ -43,6 +43,7 @@ function willCoercionThrow(value: mixed): boolean {
   }
 }
 
+/** @noinline */
```

Comment:
This is a fun one. Closure doesn't think `'' + ` can throw so it DCE this whole set of warnings. We'll have to watch out for stuff like that.

Luckily this was caught by build tests.

Reply:
Does Closure keep this comment? I remember we had a similar one in warning() but not sure if it’s gone now.